app/controllers/skins_controller.rb

```diff
@@ -218,7 +218,7 @@ def load_archive_parents
       last_position = params[:skin][:skin_parents_attributes]&.keys&.map(&:to_i)&.max || 0
       archive_parents.each do |parent_skin|
         last_position += 1
-        new_skin_parent_hash = ActionController::Parameters.new({position: last_position, parent_skin_id: parent_skin.id})
+        new_skin_parent_hash = ActionController::Parameters.new({ position: last_position, parent_skin_id: parent_skin.id, parent_skin_title: parent_skin.title })
```

Comment:
Another way to handle this would have been to modify the `reject_if` condition to allow `parent_skin_id` or `parent_skin_title`, but this should also work. (I was a bit worried about the issue of [non-unique skin titles](https://otwarchive.atlassian.net/browse/AO3-5918), but if that affected the archive parent skin titles, it would also affect them when updating a skin. So it should be okay when first adding the parents.)

Reply:
> Another way to handle this would have been to modify the `reject_if` condition to allow `parent_skin_id` or `parent_skin_title`.

Let's do that instead, and keep validations in models instead of controllers when we can.